components/Converters/samples/Converters.md

```diff
@@ -12,151 +12,71 @@ issue-id: 0
+Converts a color to the approximated display name.
 
-It can also be used to switch between two values of brush.
+> [!Sample ColorToDisplayNameConverterSample]
 
-Note : you can use a resource for the brush or pass the color string and have it converted to a brush automatically.
+## DoubleToObjectConverter
+Converts a double value into an other object. Can be used to convert doubles to e.g. visibility, colors , or different images. If `GreaterThan` and `LessThan` are both set, the logic looks for a value between the two values. Otherwise the logic looks for the value to be `GreaterThan` or `LessThan` the specified value. The `ConverterParameter` can be used to invert the logic.
 
-```xaml
-<Page.Resources>
-    <converters:BoolToObjectConverter x:Key="BoolToBrushConverter" TrueValue="Green" FalseValue="{StaticResource NopeBrush}" />
-</Page.Resources>
-```
+> [!Sample DoubleToObjectConverterSample]
 
-and using it like that :
+## DoubleToVisibilityConverter
+Converts a double value into an other object. Can be used to convert doubles to e.g. visibility, colors , or different images. If `GreaterThan` and `LessThan` are both set, the logic looks for a value between the two values. Otherwise the logic looks for the value to be `GreaterThan` or `LessThan` the specified value. The `ConverterParameter` can be used to invert the logic.
 
-```xaml
-<Border Background="{x:Bind Path=MyBoolValue, Converter={StaticResource BoolToBrushConverter}}" />
-```
+> [!Sample DoubleToVisibilityConverterSample]
 
-An other example is to switch between two images by specifying their source :
+## EmptyCollectionToObjectConverter & CollectionVisibilityConverter
+Converts a collection size into an other object. Can be used to convert to bind a visibility, a color or an image to the size of the collection. The `CollectionVisibilityConverter` is derived from it and can be easily used to convert a collection into a `Visibility` enumeration (Collapsed if the given collection is empty or null).
 
-```xaml
-<Page.Resources>
-    <converters:BoolToObjectConverter x:Key="BoolToImageConverter" TrueValue="ms-appx:///Assets/Yes.png" FalseValue="ms-appx:///Assets/No.png" />
-</Page.Resources>
-```
+> [!Sample CollectionVisibilityConverterSample]
 
-and using it like that :
+## EmptyObjectToObjectConverter, EmptyStringToObjectConverter & StringToVisibilityConverter
+Converts an object value into a an object (if the value is null returns the false value). Can be used to bind a visibility, a color or an image to the value of an object.
+The `EmptyStringToObjectConverter` converts a string value into a an object (if the value is null or empty returns the false value). Can be used to bind a visibility, a color or an image to the value of a string.
 
-```xaml
-<Image Source="{x:Bind Path=MyBoolValue, Converter={StaticResource BoolToImageConverter}}" />
-```
+> [!Sample EmptyStringToObjectConverterSample]
 
-## BoolToVisibilityConverter Examples
+## FileSizeToFriendlyStringConverter
```

Comment:
This one is missing a sample, maybe could simulates a progress bar 'download' with increasing size to a target value that goes when you click a button... That may be a bit complicated, but would make an interesting/realistic sample...

Reply:
Yeah, was thinking about this.. tried to add a slider option as a "filesize in bytes" thing.. but converting that to a `long` would require a `ConvertToLong` method in the XAML string, which might be confusing..

Added a simple sample that just parses a defined `long`:

![image](https://github.com/CommunityToolkit/Windows/assets/9866362/038e9bc1-21b5-4371-8dbe-88260cfeff93)